### ISSUE TITLE
add contributing.md

### ISSUE THREAD

**Issue by aschweigert** [publicmediaplatform/pmp-wordpress#80]
since we're starting to get people using this thing we should add a contributing.md that outlines things such as:

- how to report issues/make feature requests
- our preferred process for pull requests
- any coding/style guidelines submitted patches should adhere to

**Comment by aschweigert** [comment]
we can steal the one we use at INN (https://github.com/INN/docs/blob/master/contributing.md) or come up with something else, just figured it would be easier to outline this and have a reference than try to handle requests ad hoc

**Comment by cavis** [comment]
Agreed - and that contributing guide looks like a good starting point.  (It's the PR process I usually go with for external contributors).  #70 may also be related to this.

As far as "discussing" feature requests - would the github issue tracker be the best place for that?  There's also discuss.pmp.io - but that's currently more for PMP API discussions.

I would defer to @rnagle on coding/style.

**Comment by cavis** [comment]
I've got something in the works for this.
